test/app/model/Rook.spec.ts

```diff
@@ -87,3 +80,15 @@ describe('Rook movement', () => {
     expect(rockPossibleMoves).toIncludeSameMembers(possibleMovesWhenRookOnD4);
   });
 });
+
+const emptyBoard: Board = { onPositionPiece: (square: Square) => null };
+type SquareWithPiece = { [key: string]: Piece };
+
+const boardWithPieces: (squaresWithPiece: SquareWithPiece) => Board = (squaresWithPiece) => {
+  return {
+    onPositionPiece: (square: Square) => {
+      const pieceOnSquare = squaresWithPiece[`${square.column}${square.row}`];
+      return pieceOnSquare ? pieceOnSquare : null;
+    },
+  };
+};
```

Comment:
Czy nie lepiej wrzucić to w osobny plik w folderze z testami?

Reply:
Jak najbardziej można to zrobić - na razie było używane w jednym.

Reply:
Pyknięte :)